### PR TITLE
[MF] Creates global color, btn, font styling

### PR DESCRIPTION
**Pull request by krsnamara**
## Description

This code creates some CSS variables in the index.css file to be accessed through out the app. These variables are then applied to all HTML button elements throughout the app from the index.css page. The styles can be overwritten in particular instances at the level they need to be.

## Related Issue

related to issue #14 
closes issue #83 

## Acceptance Criteria

 [x] Create CSS variables with brand colors including hovers
 [x] Define at the index level how button inheritance style will look
 [x] Confirm that the styles are applied in the app correctly

## Type of Changes

Use one or more labels to help your team understand the nature of the change(s) you’re proposing. E.g., `bug fix` or `enhancement` are common ones.

##### Landing Page

Screenshot 2024-04-03 at 12 15 05 PM

##### Lists page

Screenshot 2024-04-03 at 12 21 11 PM

#### Manage-List Page

Screenshot 2024-04-03 at 12 22 10 PM

#### Dialog open

Screenshot 2024-04-03 at 12 22 24 PM

#### Landing page

Screenshot 2024-04-03 at 12 22 46 PM

##### Lists page

Screenshot 2024-04-03 at 12 16 26 PM

#### Manage-List Page

Screenshot 2024-04-03 at 12 17 24 PM

#### Dialog open

Screenshot 2024-04-03 at 12 19 50 PM

## Testing Steps / QA Criteria

Nothing changed but styling. To test open app, sign in and walk through the functionality to observe and test all buttons to view the painting of the app with new css variables.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -14,6 +14,12 @@
 	--color-border: hsla(220, 13%, 32%, 1);
 	--color-error: var(--color-red);
 	--color-text: var(--color-white);
+
+	--color-branding: #5a23b3;
+	--btn-confirm: var(--color-branding);
+	--btn-confirm-hover: #3f0f7f;
+	--btn-danger: var(--color-error);
+	--btn-danger-hover: #a80000;
```
Comment (mxmason) [inline src/index.css]: 🔍 some of these `--btn` variables could be renamed to suit their use case, IMO. 

For instance: what is `--btn-confirm`? Is it a background color? A text color? A font-size? We know right now that it's a color, but if `--btn-confirm` is referenced in a file far away from here, we have to do some excavation to come back here and understand what the value actually is.

**Review by mxmason — APPROVED** [review]
Looks good! I offer you a naming nit.

File: src/index.css (hunk 1 above)
Comment (krsnamara) [reply src/index.css]: fair. I am going to better name these. I really enjoy the naming nits. They all make sense after I look at the original names again with your perspective.

Comment (krsnamara) [reply src/index.css]: I pushed some edits to the naming. Let me know if I understood your suggestion correctly.

**Review by hsiangj — APPROVED** [review]
Great work!

File: src/index.css [hunk 2]
```diff
@@ -14,6 +14,14 @@
...
 	--color-error: var(--color-red);
 	--color-text: var(--color-white);
+
+	--color-branding: #5a23b3;
+	--btn-bg-color-confirm: var(--color-branding);
+	--btn-bg-color-confirm-hover: #3f0f7f;
+	--btn-bg-color-cancel: #cdcdcd;
+	--btn-bg-color-cancel-hover: #989797;
+	--btn-bg-color-delete: var(--color-error);
+	--btn-bg-color-delete-hover: #a80000;
```
Comment (mxmason) [inline src/index.css]: 🔍 I like this, but I think we could bring _even more clarity_, theoretically. These variable names describe the function of the colors, but don't tell someone at a glance what the colors are. Is `--btn-bg-color-confirm-hover` green? red? dandelion-ish?

In my time, I've seen CSS vars set up in 2 stages, with the first stage being the one that gives human-readable names to the colors, and the second stage assigning the function of the colors. I did something like this initially, although I didn't do the best job:

```css
--color-red: hsl(0, 68%, 42%);
```

We know that this variable is a red, as opposed to a blue or a green. I think it makes sense to use _that_ name when picking the background color for your button:

```css
--btn-bg-color-delete: var(--color-red);
```

All of your other colors could follow the same pattern: first name the colors, then attach them to the elements they're used in.

Comment (krsnamara) [reply src/index.css]: Ah ok I understand that now. That way when someone else looks at the line defining the --btn variable they will see a color named versus a hsl/hex/rgb. And I guess this also allows for that color to be reusable throughout the app as well?

**Pull request merged**